sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TraceConfigzZPageHandler.java

```diff
@@ -327,6 +325,44 @@ public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
     }
   }
 
+  @Override
+  public boolean processRequest(
+      String requestMethod, Map<String, String> queryMap, OutputStream outputStream) {
+    if (requestMethod.equalsIgnoreCase("POST")) {
+      try {
+        applyTraceConfig(queryMap);
+      } catch (Throwable t) {
+        try (PrintStream out = new PrintStream(outputStream, /* autoFlush= */ false, "UTF-8")) {
```

Comment:
This section seems like an error applying the config, not an error generating HTML right?

Reply:
Yea it's error applying config, I will adjust the error message